pulsar-io/rabbitmq/src/test/java/org/apache/pulsar/io/rabbitmq/sink/RabbitMQSinkTest.java

```diff
@@ -65,7 +66,15 @@ public void TestOpenAndWriteSink() throws Exception {
         RabbitMQSink sink = new RabbitMQSink();
 
         // open should success
-        sink.open(configs, null);
+        // rabbitmq service may need time to initialize
+        Awaitility.await().atMost(Duration.ofSeconds(5)).until(() -> {
+            try {
+                sink.open(configs, null);
+                return true;
```

Comment:
One way to make the code cleaner is to use `.untilAsserted` that can be used for code that isn't expected to throw exceptions.
I'd also recommend omitting `atMost(Duration.ofSeconds(5))` part. The default is 10 seconds by default and that should be fine